integrations/newrelic/newrelic_integration/utils.py

```diff
@@ -0,0 +1,34 @@
+import typing
+import jinja2
+
+from port_ocean.context.event import event
+
+from newrelic_integration.overrides import NewRelicPortAppConfig
+
+
+async def get_port_resource_configuration_by_port_kind(kind: str) -> dict:
+    app_config = typing.cast(NewRelicPortAppConfig, event.port_app_config)
+    for resource in app_config.resources:
+        if resource.kind == kind:
+            return resource.dict()
+    return {}
+
+
+async def get_port_resource_configuration_by_newrelic_entity_type(
+    entity_type: str,
+) -> dict:
+    app_config = typing.cast(NewRelicPortAppConfig, event.port_app_config)
+    for resource in app_config.resources:
+        if (
+            resource.selector.newrelic_types
+            and entity_type in resource.selector.newrelic_types
+        ):
+            return resource.dict()
```

Comment:
What if there are multiple resources matching?
U r only returning the first one

Reply:
discussed offline, will be resolved in https://getport.atlassian.net/browse/PORT-4398